src/cmo.adoc

```diff
@@ -337,8 +337,16 @@ _This specification assumes that the above constraints will typically be met for
 main memory regions and may be met for certain I/O regions._
 ====
 
-Additionally, for the purposes of PMP and PMA checks, the access size of a CMO
-instruction equals the size of the cache block accessed by the instruction.
+[NOTE]
+====
+The access size for CMO instructions is equal to the size of the cache
+block, however that access can be decomposed into multiple memory operations.
```

Comment:
I'm not sure we want to make this general of a statement.  It's clearly true that `cbo.zero` may give rise to multiple memory operations, but I don't think we want to imply that's the case for the clean/flush/inval.  Need @dkruckemyer-ventana to weigh in here.

Reply:
Ah good point, I'll change it to `cbo.zero`.

Reply:
Done, but I used a vague "in some cases" so it doesn't add any new specifications for clean/flush/inval. Would be good to know what the actual behaviour intended there is. I would guess it's the same but lets wait for @dkruckemyer-ventana .